cmd/activator/main.go

```diff
@@ -176,8 +177,20 @@ func main() {
 	reqCh := make(chan activatorhandler.ReqEvent, requestCountingQueueLength)
 	defer close(reqCh)
 
+	revDestsCh := make(chan *activatornet.RevisionDestsUpdate)
+	defer close(revDestsCh)
 	params := queue.BreakerParams{QueueDepth: breakerQueueDepth, MaxConcurrency: breakerMaxConcurrency, InitialCapacity: 0}
-	throttler := activator.NewThrottler(params, endpointInformer, sksInformer.Lister(), revisionInformer.Lister(), logger)
+
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+
+	// Start throttler
+	throttler := activatornet.NewThrottler(params, revisionInformer, endpointInformer, logger)
+	go throttler.Run(revDestsCh)
+
+	// Start revision backends manager
+	_ = activatornet.NewRevisionBackendsManager(stopCh, revDestsCh, network.AutoTransport, revisionInformer.Lister(),
+		serviceInformer.Lister(), endpointInformer, logger)
```

Comment:
If we just make this take `ctx` and use the injection stuff, then it would be like 3 parameters :)

Reply:
FWIW, I'd TODO this to get the rest baking.